test/e2e/framework/node/resource.go

```diff
@@ -518,11 +518,11 @@ func GetClusterZones(c clientset.Interface) (sets.String, error) {
 	// collect values of zone label from all nodes
 	zones := sets.NewString()
 	for _, node := range nodes.Items {
-		if zone, found := node.Labels[v1.LabelZoneFailureDomain]; found {
+		if zone, found := node.Labels[v1.LabelTopologyZone]; found {
```

Comment:
This should be beta label since line 525 also checks for LabelTopologyZone

Reply:
I'll just remove the now-redundant block